File: Python PowerPoint Examples/PropertiesApiDemo/setPptDocumentProperties.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudpowerpoint.api.properties_api.PropertiesApi(configuration)

name = "SetPptDocumentProperties.pptx" #Supported formats:ppt/pptx/pptm/pps/ppsx/pot/potx/odp
password = ""
storage = ""
folder = "input"
_property1 = spirecloudpowerpoint.models.DocumentProperty("Keywords", "Set document properties.", False)
_property2 = spirecloudpowerpoint.models.DocumentProperty("Author", "eiceblue", False)
_property3 = spirecloudpowerpoint.models.DocumentProperty("Company", "冰蓝", False)
_property4 = spirecloudpowerpoint.models.DocumentProperty("LastSavedBy", "123456@iCloud.com", False)
_property5 = spirecloudpowerpoint.models.DocumentProperty("SharedDoc", "true", False)
_list = []
_list.append(_property1)
_list.append(_property2)
_list.append(_property3)
_list.append(_property4)
_list.append(_property5)
properties = spirecloudpowerpoint.models.DocumentProperties(_list)

api.set_ppt_document_properties(name, properties=properties, password=password,
                                     storage=storage, folder=folder)

File: JavaScript PowerPoint Examples/DocumentPropertiesApiDemo/setPptDocumentProperties.js
(function (){
    var appId = "your ID";
    var appKey = "your Key";
    var baseUrl = "https://api.e-iceblue.cn";

    var SpirecloudPowerpoint = require('../../src/index');
    var apiClient = new SpirecloudPowerpoint.ApiClient(appId, appKey,baseUrl);
    var instance = new SpirecloudPowerpoint.PropertiesApi(apiClient);

    var name = "SetPptDocumentProperties.pptx";
    var properties = new SpirecloudPowerpoint.DocumentProperties();
    properties.List = new Array();
    var property1 = new SpirecloudPowerpoint.DocumentProperty();
    property1.Name = "Keywords";
    property1.Value = "Set document properties.";
    property1.BuiltIn = false;
    properties.List.push(property1);
    var property2 = new SpirecloudPowerpoint.DocumentProperty();
    property2.Name = "Author";
    property2.Value = "eiceblue";
    property2.BuiltIn = false;
    properties.List.push(property2);
    var property3 = new SpirecloudPowerpoint.DocumentProperty();
    property3.Name = "Company";
    property3.Value = "冰蓝";
    property3.BuiltIn = true;
    properties.List.push(property3);
    var property4 = new SpirecloudPowerpoint.DocumentProperty();
    property4.Name = "LastSavedBy";
    property4.Value = "123456@iCloud.com.";
    property4.BuiltIn = true;
    properties.List.push(property4);
    var property5 = new SpirecloudPowerpoint.DocumentProperty();
    property5.Name = "SharedDoc";
    property5.Value = "true";
    property5.BuiltIn = false;
    properties.List.push(property5);
    var opt = { 'folder': "input", 'properties': properties };
    var response = instance.setPptDocumentProperties(name, opt,
        function (error, data, response) {
            if (error) throw error;
        });
})();